lib/apple/index.js

```diff
@@ -74,19 +74,28 @@ function parseResult(result) {
 			productId = lastReceipt.product_id;
 			transactionId = lastReceipt.transaction_id;
 			purchaseDate = parseTime(lastReceipt.purchase_date_ms);
-			expirationDate = parseTime(lastReceipt.expires_date_ms || lastReceipt.expires_date);
+			expirationDate = parseTime(
+				lastReceipt.expires_date_ms ||
+				lastReceipt.expires_date
+			);
 		} else if (result.latest_receipt_info && result.latest_receipt_info.transaction_id) {
 			latestReceiptInfo = [result.latest_receipt_info];
 
 			productId = result.latest_receipt_info.product_id;
 			transactionId = result.latest_receipt_info.transaction_id;
 			purchaseDate = parseTime(result.latest_receipt_info.purchase_date_ms);
 			expirationDate = parseTime(
-				result.latest_receipt_info.expires_date_ms || result.latest_receipt_info.expires_date
+				result.latest_receipt_info.expires_date_ms ||
+				result.latest_receipt_info.expires_date
 			);
 		}
 	}
 
+	// Check for cancellation date
```

Comment:
Maybe a bit more information here, would be wise? Seeing as the amount of confusion exists around why this is a good idea :)